src/sentry/conf/server.py

```diff
@@ -802,6 +802,7 @@ def create_partitioned_queues(name):
     'request': 'sentry.interfaces.http.Http',
     'sdk': 'sentry.interfaces.sdk.Sdk',
     'stacktrace': 'sentry.interfaces.stacktrace.Stacktrace',
+    'frame': 'sentry.interfaces.stacktrace.Frame',
```

Comment:
if we dont special case this (i didnt see) then it means someone could send this attribute at the top level of the payload

Reply:
Isn't that already the case?

Reply:
Oh sorry, missed that. Yeah, that should not go there.

Reply:
gotcha, will fix